lib/ansible/modules/files/archive.py

```diff
@@ -142,13 +145,24 @@
 
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils._text import to_native
+from ansible.module_utils.six import PY3
+
+if PY3:
+    import lzma
```

Comment:
You'll need a `try/except ImportError` block around this to handle the case where it's not available.

Apparently our Python 3.7 build doesn't have lzma enabled, which is something I've added to my list to fix. However, we'll still want to handle this case for users that don't have it enabled.